modules/naveggIdSystem.js

```diff
@@ -11,11 +11,34 @@ import { getStorageManager } from '../src/storageManager.js';
 const MODULE_NAME = 'naveggId';
 const OLD_NAVEGG_ID = 'nid';
 const NAVEGG_ID = 'nvggid'
+const BASE_URL = 'https://id.navegg.com/uid/';
+const DEFAULT_EXPIRE = 8 * 24 * 3600 * 1000;
 
 export const storage = getStorageManager();
 
+function getNaveggIdFromApi() {
+  const resp = fetch(BASE_URL)
```

Comment:
please use methods in https://github.com/prebid/Prebid.js/blob/c5ab1dc7a890d1c541afe27509a54391247d7183/src/ajax.js